src/calendar/week.jsx

```diff
@@ -61,11 +61,14 @@ export default class Week extends React.Component {
   }
 
   _dateClasses(date) {
-    const {today, activeMonth} = this.props
+    const {today, activeMonth, selectedMax, selectedMin} = this.props
+
     return classnames({
       'is-selected': this._dateSelected(date),
       'is-today': isSameDay(today, date),
       'is-current_month': isSameMonth(date, activeMonth),
+      'is-start_selection': isSameDay(selectedMin, date),
```

Comment:
No, it's not `isRequired` (see prop types), so you must do `selectedMin && isSameDay(selectedMin, date)` and add a test case for that.

The same for `selectedMax`.